.github/workflows/main.yml

```diff
@@ -261,6 +261,7 @@ jobs:
           - TEST=demos RETURNN_DISABLE_TF=1
           - TEST=PTDataset
           - TEST=torch_frontend
+          - TEST=torch_internal_frontend
```

Comment:
As discussed, the tests should be torch independent. We should not have torch specific tests here.

Reply:
Since we need this PR merged I won't do this here, we can implement the unified tests later on.

Reply:
Ok.